www/install.rst

```diff
@@ -1,113 +1,99 @@
 ===================
-Installing packages
+Installation
 ===================
 
-These are general instructions for installing packages in the SciPy
-ecosystem.
-
-Scientific Python distributions
--------------------------------
-
-For many users, especially on Windows, the easiest way to begin is to
```

Comment:
I'd like to keep some form of this recommendation. Perhaps at the end of the new section, in italics. Something like _"If you don't know which installation method you need or prefer, we recommend you use a scientific Python distribution (see next section)"_

Reply:
Added that, but made it more specific to Anaconda, since "scientific Python distribution" itself still leaves open confusing choices.

Reply:
Sounds good, thanks!